basics/components/minion.md

```diff
@@ -220,3 +220,18 @@ After annotating the classes, put them under the package of name `org.apache.pin
 
 See [SimpleMinionClusterIntegrationTest](https://github.com/apache/incubator-pinot/blob/master/pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/SimpleMinionClusterIntegrationTest.java) where the `TestTask` is plugged-in.
 
+## Task related metrics
+
+There is a controller job runs every 5 minutes by default and emits metrics about Minion tasks scheduled in Pinot.
+For now, the following metrics are emitted for each task type:
+* ***NumMinionTasksInProgress***: Number of running tasks
+* ***NumMinionSubtasksRunning***: Number of running sub-tasks
+* ***NumMinionSubtasksWaiting***: Number of waiting sub-tasks (unassigned to a minion as yet)
+* ***NumMinionSubtasksError***: Number of error sub-tasks (completed with an error/exception)
+* ***PercentMinionSubtasksInQueue***: Percent of sub-tasks in error
```

Comment:
The description doesn't match with the meaning of the metric here.

Reply:
fixed